experimental/GaloisGrp/src/Subfields.jl

```diff
@@ -4,6 +4,11 @@ using Oscar
 using Oscar.GaloisGrp
 import Oscar.GaloisGrp: POSet, POSetElem, GaloisCtx, find_prime,
                         primitive_by_shape, bound_to_precision
+import Oscar: subfield
+
+if isdefined(Oscar, :subfield)
+  import Oscar: subfield
+end
 
 if isdefined(Oscar, :subfield)
   import Oscar: subfield
```

Comment:
this does not make sense, probably some merge artifact
```suggestion

if isdefined(Oscar, :subfield)
  import Oscar: subfield
```